patterns/pattern-1/images/analytics/build-image.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------
+#
+# Copyright 2017 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+
+# Product Context
+product=wso2ei
+version=6.1.1
+profile=analytics
+patternNumber=1
+
+# Image building - Needs to be updated with simple docker command approach
+echo "Image created: ${product}-p${patternNumber}-${profile}:${version}"
```

Comment:
Would you mind explaining the purpose of this script? As it seems like this only has an echo command.